packages/cli/src/commands/git/index.ts

```diff
@@ -96,6 +96,7 @@ export default async function main(client: Client) {
   }
 
   const { org, project } = linkedProject;
+  client.config.currentTeam = org.type === 'team' ? org.id : undefined;
```

Comment:
Is this change related?

Reply:
@TooTallNate Yes, this is used for all client calls

https://github.com/vercel/vercel/blob/5b88f673f89c44b9e91e44247e29be6bed43805a/packages/cli/src/util/client.ts#L99